### PR TITLE
Exercises for sections of numbers and parameters. Update of knowledge section

### PR DESCRIPTION
**Pull request by BakanovskayaNastya**
… section

### CODE REVIEW

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/content_html/def_subjdomain_numbers_and_numeric_structures_exercise_1.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<p>Представьте следующее утверждение на языке Scg: "Отрезок DM равен одной третьей отрезка DE."</p>
```
Comment (deniskoronchik) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/content_html/def_subjdomain_numbers_and_numeric_structures_exercise_1.html]: Most of file name already exist in a file path (directories name). I think it should be shorter

File: ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_knowledges/section_subjdomain_knowledges.scsi [hunk 2]
```diff
@@ -656,3 +657,75 @@ nrel_detalization
...
+			exercise
+		};;
+	*);
+<- rrel_key_sc_element: ...
+	(*
+		<- explanation;;
+		<= nrel_sc_text_translation:
+		...
+		(*
+		-> rrel_example:
```
Comment (deniskoronchik) [inline ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_knowledges/section_subjdomain_knowledges.scsi]: No good tabulation

File: ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_knowledges/section_subjdomain_knowledges.scsi [hunk 3]
```diff
@@ -656,3 +657,75 @@ nrel_detalization
...
+				"file://content_html/explanation_for_exercise.html"
```
Comment (deniskoronchik) [inline ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_knowledges/section_subjdomain_knowledges.scsi]: No good tabulation

File: ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_knowledges/section_subjdomain_knowledges.scsi [hunk 4]
```diff
@@ -656,3 +657,75 @@ nrel_detalization
...
+		};;
+	*);
+<- rrel_key_sc_element: ...
+	(*
+		<- explanation;;
+		<= nrel_sc_text_translation:
+		...
+		(*
+		-> rrel_example:
+				"file://content_html/explanation_for_right_answer.html"
```
Comment (deniskoronchik) [inline ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_knowledges/section_subjdomain_knowledges.scsi]: bad tabs

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs [hunk 5]
```diff
@@ -8,21 +8,10 @@ section_subjdomain_numbers_and_numeric_structures
...
-	...
-(*
-	<= nrel_sc_text_translation:
-	 ...
-	 (*
-		 -> rrel_example:
-		   "file://content_html/introduction_section_subjdomain_numbers.pdf"
-			(* <- lang_ru;;	*);;
-	 *);;
-*);
```
Comment (ShunkevichDV) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs]: What was wrong with that part? It is important, why was it deleted?

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/nl_files/def_section_subjdomain_parameters_and_values_exercise_2.html [hunk 6]
```diff
@@ -0,0 +1 @@
+<p>Представьте следующее утверждение на языке Scg: "Возможное отклонение измерения нормальной температуры тела равно 1 градус Цельсия."</p>
```
Comment (Besomhead) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/nl_files/def_section_subjdomain_parameters_and_values_exercise_2.html]: c in SCg should be capital

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs [hunk 7]
```diff
@@ -44,3 +44,91 @@ section_subjdomain_parameters_and_values
...
+	(*
+	<- rrel_key_sc_element: 
+		...
+ 		(*
+  		<- illustration;;
+  		<= nrel_sc_text_translation:
+  			...
+  			(*
+  			-> rrel_example:
+    				"file://nl_files/exercise_parameters_1_right_answer.png"
```
Comment (Besomhead) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs]: Why are they variable?

File: ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_knowledges/content_html/explanation_for_right_answer.html [hunk 8]
```diff
@@ -0,0 +1 @@
+<p><b><sc_element sys_idtf  = "nrel_right_answer">правильный ответ</sc_element></b> - би<sc_element sys_idtf  = "binary_relation">бинарное</sc_element> <sc_element sys_idtf  = "oriented_relation">ориентированное отношение</sc_element>, связывающее некоторое <sc_element sys_idtf  = "exercise">упражнение</sc_element> к <sc_element sys_idtf  = "section">раздел базы знаний</sc_element> и некоторый <sc_element sys_idtf  = "file">файл</sc_element>, являющийся правильным ответом к нему.</p>
```
Comment (Besomhead) [inline ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_knowledges/content_html/explanation_for_right_answer.html]: misspelling

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs [hunk 9]
```diff
@@ -44,3 +44,91 @@ section_subjdomain_parameters_and_values
...
+=>nrel_exercise: 
+ section_subjdomain_parameters_and_values_exercise_1;; 
+
+section_subjdomain_parameters_and_values_exercise_1
+<- exercise;
+
+=> nrel_main_idtf:
+ 	[Раздел. Предметная область параметров и величин. Упражнение 1]
+ 	(* <- lang_ru;; *);
+ 	[Section. Subject domain of parameters and values. Exercise 1]
```
Comment (Besomhead) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs]: I think "Exercise n" should be at the first place

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs [hunk 10]
```diff
@@ -44,3 +44,91 @@ section_subjdomain_parameters_and_values
 	nrel_large_quantitie;
 	nrel_equal_of_quantities;
 	nrel_greater_or_equal_quantitie;;
+
+section_subjdomain_parameters_and_values
+=>nrel_exercise: 
+ section_subjdomain_parameters_and_values_exercise_1;;
```
Comment (Besomhead) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs]: I think exercise_n should be at the first place

File: ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_knowledges/section_subjdomain_knowledges.scsi [hunk 11]
```diff
@@ -5,8 +5,8 @@ subject_domain_of_knowledges
 	[Subject domain of knowledges]	
 	(* <- lang_en;; *);
 <- subject_domain;
-<- sc_node_struct;
-<- structure;
+-<- sc_node_struct;
```
Comment (Besomhead) [inline ims/ostis_tech/unificated_models/section_unificated_models_kb/section_subjdomain_knowledges/section_subjdomain_knowledges.scsi]: lexer error

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs (hunk 5 above)
Comment (Besomhead) [reply ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs]: We still need it.

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs [hunk 12]
```diff
@@ -55,3 +43,130 @@ section_subjdomain_numbers_and_numeric_structures
...
+	<- rrel_key_sc_element: 
+		...
+		(*
+		<- illustration;;
+		<= nrel_sc_text_translation:
+		  	... 
+		  	(*
+		  	-> rrel_example:
+		    		"file://content_img/exercise_numbers_1_right_answer.png"  
+				(* <- lang_ru;; *);;
```
Comment (Besomhead) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs]: sc.g-text is not an element of russian language texts set

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs [hunk 13]
```diff
@@ -55,3 +43,130 @@ section_subjdomain_numbers_and_numeric_structures
...
+	<- rrel_key_sc_element: 
+		...
+	 	(*
+	  	<- illustration;;
+	  	<= nrel_sc_text_translation:
+	  		...
+	  		(*
+	  		-> rrel_example:
+	    			"file://content_img/exercise_numbers_2_right_answer.png"  
+			(* <- lang_ru;; *);;
```
Comment (Besomhead) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs]: sc.g-text is not an element of russian language texts set

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs [hunk 14]
```diff
@@ -55,3 +43,130 @@ section_subjdomain_numbers_and_numeric_structures
...
+	<- rrel_key_sc_element: 
+		...
+ 		(*
+		<- illustration;;
+  		<= nrel_sc_text_translation:
+  			...
+  			(*
+  			-> rrel_example:
+    				"file://content_img/exercise_numbers_3_right_answer.png"  
+				(* <- lang_ru;; *);;
```
Comment (Besomhead) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/section_subjdomain_numbers_and_numeric_structures.scs]: sc.g-text is not an element of russian language texts set

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs [hunk 15]
```diff
@@ -44,3 +44,91 @@ section_subjdomain_parameters_and_values
...
+	<- rrel_key_sc_element: 
+		...
+ 		(*
+  		<- illustration;;
+  		<= nrel_sc_text_translation:
+  			...
+  			(*
+  			-> rrel_example:
+    				"file://nl_files/exercise_parameters_1_right_answer.png"  
+				(* <- lang_ru;; *);;
```
Comment (Besomhead) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs]: sc.g-text is not an element of russian language texts set

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs (hunk 7 above)
Comment (Besomhead) [reply ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs]: Still need to be corrected.

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs [hunk 16]
```diff
@@ -44,3 +44,91 @@ section_subjdomain_parameters_and_values
...
+	<- rrel_key_sc_element: 
+		...
+ 		(*
+  		<- illustration;;
+  		<= nrel_sc_text_translation:
+  			...
+  			(*
+  			-> rrel_example:
+    				"file://nl_files/exercise_parameters_2_right_answer.png"  
+				(* <- lang_ru;; *);;
```
Comment (Besomhead) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs]: sc.g-text is not an element of russian language texts set

File: ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs [hunk 17]
```diff
@@ -44,3 +44,89 @@ section_subjdomain_parameters_and_values
...
+	(*
+	<- rrel_key_sc_element: 
+		...
+ 		(*
+  		<- illustration;;
+  		<= nrel_sc_text_translation:
+  			...
+  			(*
+  			-> rrel_example:
+    				"file://nl_files/exercise_parameters_1_right_answer.png";;
```
Comment (Besomhead) [inline ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_parameters_and_values/section_subjdomain_parameters_and_values.scs]: wrong_arcs
Check the arcs.

**Pull request merged**